config/secrets.yml

```diff
@@ -0,0 +1,22 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rake secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
+  secret_key_base: 9f147bb0e4e15668c47f2d79ae6114442270689156132e913d59c262350703eb02a447c9bb09bd07a3b8049f030158cd26ac180cda839baf0f67ff9158a75b28
```

Comment:
I'd be a fan of using `<%= SecureRandom.hex %>` if that will work, that way no one is tempted to use a default key that's shared publicly